diffsims/crystallography/reciprocal_lattice_vector.py

```diff
@@ -1128,6 +1135,8 @@ def from_min_dspacing(cls, phase, min_dspacing=0.7):
         dspacing = 1 / phase.structure.lattice.rnorm(hkl)
         idx = dspacing >= min_dspacing
         hkl = hkl[idx]
+        if include_zero_vector:
```

Comment:
Can we add a test for this change?

Reply:
Yeah, I was kind of surprised that didn't cause an error with the test suite.